engine/gui/src/gui.cpp

```diff
@@ -2770,10 +2771,42 @@ namespace dmGui
         scale = mulPerElem(adjust_scale, scale);
     }
 
+    // Same euler checks as in gameobject.cpp
```

Comment:
Now we do the same check as game objects do, since we have two properties controlling the same rotation.